libcxx/CMakeLists.txt

```
@@ -177,6 +177,10 @@ set(LIBCXX_LIBDIR_SUFFIX "${LLVM_LIBDIR_SUFFIX}" CACHE STRING
     "Define suffix of library directory name (32/64)")
 option(LIBCXX_INSTALL_HEADERS "Install the libc++ headers." ON)
 option(LIBCXX_INSTALL_LIBRARY "Install the libc++ library." ON)
+cmake_dependent_option(LIBCXX_INSTALL_MODULES
+  "Install the libc++ C++20 modules (experimental)." OFF
+  "LIBCXX_ENABLE_STD_MODULES" OFF
+)
```

The terminology used here is unclear regarding what is being installed. "C++20 modules" is a language feature and thus not something that is installable. What is being installed is source code files for the module interface units.
```suggestion
cmake_dependent_option(LIBCXX_INSTALL_MODULE_INTERFACE_UNIT_SOURCES
  "Install the libc++ C++20 module interface unit source files (experimental)." OFF
  "LIBCXX_ENABLE_STD_MODULES" OFF
)
```